config/samples/custom-rules/openshift-virtualization/kubevirt-enforce-trusted-tls-registries.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: compliance.openshift.io/v1alpha1
+kind: CustomRule
+metadata:
+  name: kubevirt-enforce-trusted-tls
+  namespace: openshift-compliance
+spec:
+  title: "Only trusted registries using TLS can be used"
+  id: kubevirt_enforce_trusted_tls
+  description: |-
+    By only pulling container images from trusted registries, organizations 
+    can reduce the risk of introducing unknown vulnerabilities or malicious 
+    software into their systems. This helps ensure that their applications and systems 
+    remain secure and stable.
+  failureReason: |-
+    There are registries not using TLS in '.spec.storageImport.insecureRegistries' in
+    the 'kubevirt-hyperconverged' resource.
+  severity: Medium
+  checkType: Platform
+  scannerType: CEL
+  inputs:
+    - name: hcoList
+      kubernetesInputSpec:
+        apiVersion: hco.kubevirt.io/v1beta1
+        resource: hyperconvergeds
```

Comment:
We can simplify this rule if we only look for the hyperconverged resource (since I just learned that there can/should only be one in the cluster  😄 )

```yaml
inputs:
    - name: hcoList
      kubernetesInputSpec:
        apiVersion: hco.kubevirt.io/v1beta1
        resource: hyperconvergeds
        resourceName: kubevirt-hyperconverged
        resourceNamespace: openshift-cnv
```